src/chttpd/src/chttpd_util.erl

```diff
@@ -95,3 +96,20 @@ handle_legacy_config(OriginalHeaders, DefaultHeaderValue) ->
         false ->
             OriginalHeaders
     end.
+
+get_dbs_info_shards(DbName) ->
```

Comment:
The name `_shards` seems misleading here. We're not getting individual shards here but the already aggregated db info for the whole (clustered) db.

Reply:
Changed to `get_dbs_info_helper`, thanks for the review.